src/Components/IncidentList/IncidentCard.test.js

```diff
@@ -1,28 +1,32 @@
 import React from 'react';
-import { configure, shallow, mount } from 'enzyme';
-import sinon from 'sinon';
-import Adapter from 'enzyme-adapter-react-16/build';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
 import IncidentCard from './IncidentCard.Component';
 
 configure({ adapter: new Adapter() });
 
 describe('Incident Card', () => {
   let wrapper;
   let props;
-  const renderFlag = sinon.spy();
   beforeEach(() => {
     props = {
-      incidentId: '',
-      incidentSubject: '',
+      incidentId: '1',
+      incidentSubject: 'Harassment',
       incidentDescription: '',
       incidentReportDate: '',
       incidentTime: '',
-      incidentFlag: '',
+      incidentFlag: 'Pending',
       assignees: [],
     };
+    if (wrapper) wrapper.unmount();
   });
   it('should render all div elements', () => {
     wrapper = shallow(<IncidentCard {...props} />);
     expect(wrapper.find('div').length).toEqual(9);
   });
+
+  it('should render assignee', () => {
+    wrapper = shallow(<IncidentCard {...props} />);
```

Comment:
I think the `wrapper` variable should be moved to `beforeEach` to avoid repetition. Your thoughts?

Reply:
it is not recommended. Tests will mostly fail because they will be rendered before the actual event